### PR TITLE
fixing some specifics in the readme

### PR DESCRIPTION
**Pull request by arendej**

### CODE REVIEW

File: examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Prior to creating a ROSA STS cluster, you must create the required account-wide
         ```
         export TF_VAR_token=<ocm_offline_token>
         ```
-    3.  This value should point to your OpenShift instance.  
+    3.  This value should always point to `https://api.openshift.com` (unless you're a Red Hat engineer and you know of a different one ;) ).
```
Comment (bardielle) [inline examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md:26]: That description is also copied in other README files (for example github IDP)  would you prefer to change it in all other places?

Comment (arendej) [reply examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md:26]: Yes, please!

File: README.md [hunk 2]
```diff
@@ -83,5 +83,5 @@ If you want to locally test the provider binary without building from sources, y
     
 to your local using the following example: 
 
-    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.1/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
+    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-e2e:latest cat /root/.terraform.d/plugins/terraform.local/local/ocm/1.0.4/linux_amd64/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```
Comment (osherdp) [inline README.md:86]: if you want to make it even leaner and without the need of specifying the version:
```suggestion
    podman run --pull=always --rm registry.ci.openshift.org/ci/ocm-tf-bin:latest cat /root/terraform-provider-ocm > ~/terraform-provider-ocm && chmod +x ~/terraform-provider-ocm
```

``ocm-tf-bin`` is a new image we're publishing which is much smaller in size (219MB vs 2.18GB)

File: examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md (hunk 1 above)
Comment (bardielle) [reply examples/create_rosa_cluster/create_rosa_sts_cluster/classic_sts/account_roles/README.md:26]: Didn't forget about it - I'm waiting till the renaming changes will be merged

**Comment by bardielle** [comment]
I opened a PR (https://github.com/terraform-redhat/terraform-provider-rhcs/pull/149) that fixed more changes and it was merged. 
I will close that PR

**Pull request closed without merging**